### PR TITLE
[LLM] Llama 3.2 guide

### PR DESCRIPTION
**Pull request by Michaelvll**
Point, launch and serve vision llama 3.2

Tested (run the relevant ones):

- [ ] Code formatting: `bash format.sh`
- [ ] Any manual or new tests for this PR (please specify below)
  - [x] `sky launch ./llama3_2.yaml --env HF_TOKEN --cloud gcp` with `curl` requests
  - [x] `sky launch -c llama3_2_vision --cloud gcp llama3_2-vision-90b.yaml --use-spot --env HF_TOKEN` with `curl`
- [ ] All smoke tests: `pytest tests/test_smoke.py` 
- [ ] Relevant individual smoke tests: `pytest tests/test_smoke.py::test_fill_in_the_name` 
- [ ] Backward compatibility tests: `conda deactivate; bash -i tests/backward_compatibility_tests.sh`

### CODE REVIEW

File: docs/source/_static/custom.js [hunk 1]
```diff
@@ -31,6 +31,7 @@ document.addEventListener('DOMContentLoaded', () => {
         { selector: '.toctree-l1 > a', text: 'Pixtral (Mistral AI)' },
         { selector: '.toctree-l1 > a', text: 'Many Parallel Jobs' },
         { selector: '.toctree-l1 > a', text: 'Reserved, Capacity Blocks, DWS' },
+        { selector: '.toctree-l1 > a', text: 'Llama-3.2 (Meta)' },
```
Comment (concretevitamin) [inline docs/source/_static/custom.js:34]: Add admin policy to new too?

File: llm/llama-3_2/README.md [hunk 2]
```diff
@@ -0,0 +1,356 @@
...
+
+
+## Prerequisites
+
+- Go to the [HuggingFace model page](https://huggingface.co/meta-llama/) and request access to the model [meta-llama/Llama-3.2-1B-Instruct](https://huggingface.co/meta-llama/Llama-3.2-1B-Instruct) and [meta-llama/Llama-3.2-11B-Vision](https://huggingface.co/meta-llama/Llama-3.2-11B-Vision).
+- Check that you have installed SkyPilot ([docs](https://skypilot.readthedocs.io/en/latest/getting-started/installation.html)).
+- Check that `sky check` shows clouds or Kubernetes are enabled.
+
+## SkyPilot YAML
+
```
Comment (concretevitamin) [inline llm/llama-3_2/README.md:34]: Maybe add a quick table/bullet list of all Llama 3.2 variants, and hint users to replace `envs.MODEL_NAME`.

File: llm/llama-3_2/README.md [hunk 3]
```diff
@@ -0,0 +1,356 @@
...
+* [Llama 3.2 release](https://ai.meta.com/blog/llama-3-2-connect-2024-vision-edge-mobile-devices/)
+
+
+
+## Why use SkyPilot?
+
+* Point, launch, and serve: simply point to the cloud/kubernetes cluster you have access to, and launch the model there with a single command.
+* No lock-in: run on any supported cloud - AWS, Azure, GCP, Lambda Cloud, IBM, Samsung, OCI
+* Everything stays in your cloud account (your VMs & buckets)
```
Comment (concretevitamin) [inline llm/llama-3_2/README.md:19]:
```suggestion
* **Point, launch, and serve**: simply point to the cloud/Kubernetes cluster you have access to, and launch the model there with a single command.
* No lock-in: run on any supported cloud — AWS, Azure, GCP, Lambda Cloud, IBM, Samsung, OCI
* Everything stays in your cloud account (your VMs & buckets)
```

File: llm/llama-3_2/README.md [hunk 4]
```diff
@@ -0,0 +1,356 @@
+
+# Vision Llama-3.2 (Meta) 
+
+
+<p align="center">
+<img src="https://i.imgur.com/1NEZs9f.png" alt="Llama-3 x SkyPilot" style="width: 50%;">
+</p>
+
+[Llama 3.2](https://ai.meta.com/blog/llama-3-2-connect-2024-vision-edge-mobile-devices/) is out on the date of Sep 25th 2024. It not only includes the latest models for chat, but also includes the multimodal models. Let's point and launch it with SkyPilot.
```
Comment (concretevitamin) [inline llm/llama-3_2/README.md:9]:
```suggestion
[Llama 3.2](https://ai.meta.com/blog/llama-3-2-connect-2024-vision-edge-mobile-devices/) family was released by Meta on Sep 25, 2024. It not only includes the latest improved (and smaller) LLM models for chat, but also includes multimodal vision-language models. Let's _point and launch_ it with SkyPilot.
```

File: llm/llama-3_2/README.md [hunk 5]
```diff
@@ -0,0 +1,356 @@
...
+  cpus: 8+
+  disk_size: 512  # Ensure model checkpoints can fit.
+  disk_tier: best
+  ports: 8081  # Expose to internet traffic.
+
+setup: |
+  # Install huggingface transformers for the support of Llama 3.2
+  pip install git+https://github.com/huggingface/transformers.git@f0eabf6c7da2afbe8425546c092fa3722f9f219e
+  pip install vllm==0.6.2
+
```
Comment (concretevitamin) [inline llm/llama-3_2/README.md:68]:
```suggestion
```

File: llm/llama-3_2/README.md [hunk 6]
```diff
@@ -0,0 +1,356 @@
...
+To shut down all resources:
+```console
+sky serve down llama3
+```
+
+See more details in [SkyServe docs](https://skypilot.readthedocs.io/en/latest/serving/sky-serve.html).
+
+
+## Development and Finetuning Llama 3 series
```
Comment (concretevitamin) [inline llm/llama-3_2/README.md:354]:
```suggestion
## Developing and Finetuning Llama 3 series
```

**Review by concretevitamin — APPROVED** [review]
LGTM

**Comment by concretevitamin** [comment]
Maybe add to News and remove some older entries.

File: docs/source/_static/custom.js (hunk 1 above)
Comment (Michaelvll) [reply docs/source/_static/custom.js:34]: Added in #3968 . Let's keep this PR clean : )

**Pull request merged**